packages/ChangesReloaded-Core.package/CRBrowser.class/instance/displayedChangeListItems.st

```diff
@@ -0,0 +1,4 @@
+accessing
+displayedChangeListItems
+
+	^ self selectedHistoryListItem history
```

Comment:
Hmm. In the method above, `displayedChangeDescriptions`, we guard against `selectedHistoryListItem` possibly being `nil`. This would throw an error here.
Fair, for now, the method is only called if that's not the case, but it seems a bit strange to me that two methods that seem so similar from the outside behave so different in edge cases (throwing error vs returning an empty collection).